src/main/java/de/retest/recheck/review/ignore/ValuePatternFilter.java

```diff
@@ -0,0 +1,67 @@
+package de.retest.recheck.review.ignore;
+
+import java.io.Serializable;
+import java.util.Optional;
+import java.util.regex.MatchResult;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import de.retest.recheck.ignore.Filter;
+import de.retest.recheck.review.ignore.io.RegexLoader;
+import de.retest.recheck.ui.descriptors.Element;
+import de.retest.recheck.ui.diff.AttributeDifference;
+import lombok.Getter;
+
+@Getter
+public class ValuePatternFilter implements Filter {
+
+	private final Pattern pattern;
+
+	public ValuePatternFilter( final String regex ) {
+		pattern = Pattern.compile( regex );
+	}
+
+	@Override
+	public boolean matches( final Element element ) {
+		return false;
+	}
+
+	@Override
+	public boolean matches( final Element element, final AttributeDifference attributeDifference ) {
+		final Serializable actual = attributeDifference.getActual();
+
+		if ( actual == null || !(actual instanceof String) ) {
+			return false;
+		}
+
+		final Matcher matcher = pattern.matcher( (String) actual );
+
+		if ( matcher.matches() ) {
```

Comment:
Should be safe to return `matcher.matches()`

Reply:
Thanks, Done